qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -103,7 +103,7 @@ def __init_subclass__(cls, **kwargs):
     # pylint: disable=super-init-not-called
     def __init__(
         self,
-        calibrations: Calibrations,
+        calibrations: BackendCalibrations,
```

Comment:
Is there any use case for `Calibrations` as a separate class if all the calibration experiments use `BackendCalibrations`?

Reply:
I'm starting to see less of a use for it. The only one that I still see is development convenience. The Calibrations class has a fair amount of functionality that could be used in another context. My rational for keeping it separate from BackendCalibrations is to avoid too much coupling with the backend objects. Although, perhaps we should just merge the functionality of `BackendCalibrations` into `Calibrations` ?

Reply:
We don't have to do it here, but it is something to consider. If there is no independent case for `Calibrations`, it would be more straightforward to have one class than two. It does seem like `Calibrations` is an object for tracking changes to the defaults of a backend.